sdk/core/core-util/package.json

```diff
@@ -33,7 +33,7 @@
     "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
     "test:node": "npm run unit-test:node && npm run integration-test:node",
     "test": "npm run test:node && npm run test:browser",
-    "unit-test:browser": "vitest -c vitest.browser.config.mts",
```

Comment:
I noticed that there is a mismatch between the versions of playwright found in rush top-level's hooks and the ones needed here. As such, we needed to pin the version of playwright. 

As the one who pushed for this change, I now worry that it wasn't the best place for the playwright installation - how do folks feel about failing fast on that idea, inlining the playwright install in the two places its used, and I can create an issue to holistically handle this across our repo?

Reply:
Yes I noticed the mismatch and pin it in global install temporarily.  Similar to how we download test-proxy, we could include logic to install browsers in a dev-tool test command. I played with it last week in https://github.com/Azure/azure-sdk-for-js/compare/main...jeremymeng:azure-sdk-for-js:devtool/ensure-playwright-chrome#diff-da59a71042efbe9bc2e4d059cc90fbb8e64f19b0f2254fc60b7cc8bed36870e6. We will need to add a command for vitest to start test proxy anyway.